conftest.py

```diff
@@ -236,8 +231,11 @@ def sr_disk(request, host):
     yield disk
 
 @pytest.fixture(scope='session')
-def sr_disk_4k(request, host):
-    disk = request.param
+def sr_disk_4k(pytestconfig, host):
+    disks = pytestconfig.getoption("sr_disk")
```

Comment:
Should be `sr_disk_4k`. It's a different parameter than normal `sr_disk`.

Reply:
right, fixed